modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js

```diff
@@ -42,6 +42,17 @@ const CreationMenu = ({
 		primaryItems.length + secondaryItemsCountRef.current
 	);
 
+	const getPlusIconLabel = () => {
+		const item =
+			primaryItems && primaryItems[0]
+				? primaryItems[0]
+				: secondaryItems[0].items
+				? secondaryItems[0].items[0]
+				: secondaryItems[0];
```

Comment:
Just my 2 cents: this looks too concise, maybe for legibility in the future you could make it easier to read and follow what's going on. Not sure if my logic is spot on below but something in that vein.

```js
const firstPrimaryItem = primaryItems && primaryItems[0];
const firstSecondaryItem = secondaryItems[0].items ? secondaryItems[0].items[0] : secondaryItems[0];

const item = firstPrimaryItem || firstSecondaryItem;
```
![](https://media3.giphy.com/media/3og0IzoPfRVwyxjDUs/giphy.gif)

Reply:
Good point @kresimir-coko, I just copied from the v2 (https://github.com/liferay/clay/blob/2.x/packages/clay-management-toolbar/src/ClayManagementToolbar.soy#L496) but that is for sure more legible.

Reply:
Updated in  https://github.com/liferay-frontend/liferay-portal/pull/852/commits/4b524e02a25bcebf0ad1c14714727eab088243c4

Reply:
Another option just for giggles:

```js
const item =
  primaryItems?.[0] ??
  secondaryItems[0].items?.[0] ??
  secondaryItems[0];
```

(Using `??` on the assumption that we're dealing with non-empty strings, but `||` would also be a viable option, depending on the context.)